packages/terra-clinical-result-view/src/ResultView.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconDocuments from 'terra-icon/lib/icon/IconDocuments';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import { elementType } from 'react-prop-types';
+import styles from './ResultView.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   *  An array of one or more results.
+   */
+  results: PropTypes.arrayOf(PropTypes.shape({
+    value: PropTypes.number,
+    unit: PropTypes.string,
+    icon: elementType('svg'),
+    color: PropTypes.string,
+  })),
+  /**
+   *  The age display of the result. Does not handle formatting.
+   */
+  timeDisplay: PropTypes.string,
+  /**
+   *  Does the result have related documents. Adds an indicator.
+   */
+  hasDocument: PropTypes.bool,
+  /**
+   *  Has the result been modified. Adds an indicator.
+   */
+  isModified: PropTypes.bool,
+  /**
+   *  Are there comments related to the result. Adds an indicator.
+   */
+  hasComment: PropTypes.bool,
+  /**
+   *  The alignment of the component. Defaults to left.
+   */
+  alignment: PropTypes.oneOf(['right', 'center']),
```

Comment:
did you mean left instead of center?

Reply:
Nope, I mean center. We don't need to specify left explicitly since it's the default alignment. I think it'll be a good idea to support both center and right.

Reply:
I don't see that pattern being applied in other places. If a consumer wants to pass in 'left' explicitly, they should be able to.

https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-item-view/src/ItemView.jsx#L26
https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-item-view/src/ItemView.jsx#L57

Reply:
Changed it